pkg/builtinchecks/yamls/dangling-servicemonitor.yaml

```diff
@@ -0,0 +1,7 @@
+name: "dangling-servicemonitor"
+description: "Indicates when a service monitor's label selector doesn't match any pods"
```

Comment:
I think we need more context and links to indicate it's only for prometheus-operator

Reply:
added more info in commit c1ef01f, let me know if you think more is needed